tests/integration/golang/mlflow/run/create_with_namespace_test.go

```diff
@@ -0,0 +1,192 @@
+	})
+	assert.Nil(s.T(), err)
+
+	// set namespace default experiment.
+	namespace.DefaultExperimentID = experiment.ID
+	_, err = s.NamespaceFixtures.UpdateNamespace(context.Background(), namespace)
+	assert.Nil(s.T(), err)
+
+	// send request with "0" for experiment ID, which is default provided by mlflow client.
+	req := request.CreateRunRequest{
+		Name: "TestRun",
+		Tags: []request.RunTagPartialRequest{
+			{
+				Key:   "key1",
+				Value: "value1",
+			},
+			{
+				Key:   "key2",
+				Value: "value2",
+			},
+		},
+		StartTime:    1234567890,
+		ExperimentID: "0",
+	}
+
+	resp := response.CreateRunResponse{}
+	assert.Nil(
+		s.T(),
+		s.MlflowClient.WithMethod(
+			http.MethodPost,
+		).WithRequest(
+			req,
+		).WithResponse(
+			&resp,
+		).WithNamespace(
+			"custom-ns",
+		).DoRequest(
+			fmt.Sprintf("%s%s", mlflow.RunsRoutePrefix, mlflow.RunsCreateRoute),
+		),
+	)
+	assert.NotEmpty(s.T(), resp.Run.Info.ID)
+	assert.NotEmpty(s.T(), resp.Run.Info.UUID)
+	assert.Equal(s.T(), "TestRun", resp.Run.Info.Name)
+	assert.Equal(s.T(), fmt.Sprintf("%d", *experiment.ID), resp.Run.Info.ExperimentID)
+	assert.Equal(s.T(), int64(1234567890), resp.Run.Info.StartTime)
+	assert.Equal(s.T(), int64(0), resp.Run.Info.EndTime)
+	assert.Equal(s.T(), string(models.StatusRunning), resp.Run.Info.Status)
+	assert.NotEmpty(s.T(), resp.Run.Info.ArtifactURI)
+	assert.Equal(s.T(), string(models.LifecycleStageActive), resp.Run.Info.LifecycleStage)
+	assert.Equal(s.T(), []response.RunTagPartialResponse{
+		{
+			Key:   "key1",
+			Value: "value1",
+		},
+		{
+			Key:   "key2",
+			Value: "value2",
+		},
+	}, resp.Run.Data.Tags)
+}
+
+func (s *CreateRunWithNamespaceTestSuite) Test_Error() {
+	defer func() {
+		assert.Nil(s.T(), s.NamespaceFixtures.UnloadFixtures())
+	}()
+
+	namespace, err := s.NamespaceFixtures.CreateNamespace(context.Background(), &models.Namespace{
+		ID:                  1,
+		Code:                "custom-ns",
+		DefaultExperimentID: common.GetPointer(int32(0)),
+	})
+	assert.Nil(s.T(), err)
+
+	experiment, err := s.ExperimentFixtures.CreateExperiment(context.Background(), &models.Experiment{
+		Name:           uuid.New().String(),
+		NamespaceID:    namespace.ID,
+		LifecycleStage: models.LifecycleStageActive,
+	})
+	assert.Nil(s.T(), err)
+
+	// set namespace default experiment.
+	namespace.DefaultExperimentID = experiment.ID
+	_, err = s.NamespaceFixtures.UpdateNamespace(context.Background(), namespace)
+	assert.Nil(s.T(), err)
+	nonExistingExperimentID := *experiment.ID + 1
+
+	tests := []struct {
+		name      string
+		error     *api.ErrorResponse
+		namespace string
+		request   request.CreateRunRequest
+	}{
+		{
+			name:      "CreateRunWithNotExistingNamespaceAndExistingExperimentID",
+			namespace: "custom-ns1",
+			request: request.CreateRunRequest{
+				ExperimentID: fmt.Sprintf("%d", *experiment.ID),
+			},
+			error: api.NewResourceDoesNotExistError(
+				`unable to find namespace with code: custom-ns1`,
+			),
+		},
```

Comment:
can you please remove this test case from here? Ill add it in scope of my tests related to `namespace` integration tests.

Reply:
keeping for now